examples/prediction/HousingTrain.py

```diff
@@ -18,8 +18,11 @@
 import pandas as pd
 from sklearn.metrics import mean_absolute_error
 from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import Imputer
 from xgboost import XGBRegressor
+try:
+    from sklearn.preprocessing import Imputer
+except ImportError:
+     from sklearn.impute import SimpleImputer as Imputer
```

Comment:
please fix indentation here at line 25. Its generating lint error as well.

Reply:
Thanks for reivew, updated it, sorry for my typo.